docs/api/task.md

```diff
@@ -0,0 +1,109 @@
+<!-- front-matter
+id: api-task
+title: task()
+hide_title: true
+sidebar_label: task()
+-->
+
+# task()
+
+**Reminder**: This API isn't the recommended pattern anymore - export your tasks.
```

Comment:
Should there be API docs for exporting tasks? Should it be linked to "Creating Tasks" for that? Should it be left alone?

Reply:
Maybe a brief example would be enough?

Reply:
I think we're going to link to Creating Tasks in Getting Started for now. It has the best explanation.